sdk/videoanalyzer/azure-mgmt-videoanalyzer/azure/mgmt/videoanalyzer/models/_video_analyzer_enums.py

```diff
@@ -6,27 +6,11 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-from enum import Enum, EnumMeta
-from six import with_metaclass
-
-class _CaseInsensitiveEnumMeta(EnumMeta):
-    def __getitem__(self, name):
-        return super().__getitem__(name.upper())
-
-    def __getattr__(cls, name):
-        """Return the enum member matching `name`
-        We use __getattr__ instead of descriptors or inserting into the enum
-        class' __dict__ in order to support `name` and `value` being both
-        properties for enum members (which live in the class' __dict__) and
-        enum members themselves.
-        """
-        try:
-            return cls._member_map_[name.upper()]
-        except KeyError:
-            raise AttributeError(name)
-
-
-class AccessPolicyEccAlgo(with_metaclass(_CaseInsensitiveEnumMeta, str, Enum)):
+from enum import Enum
+from azure.core import CaseInsensitiveEnumMeta
+
+
+class AccessPolicyEccAlgo(str, Enum, metaclass=CaseInsensitiveEnumMeta):
```

Comment:
nit: this package is deprecated so we don't need care about it
![image](https://github.com/Azure/azure-sdk-for-python/assets/70930885/0226d03b-437f-4ab1-ac9b-b2fa7bd495ad)